frontend/apps/filemanager/filemanagershortcuts.lua

```diff
@@ -88,55 +88,55 @@ function FileManagerBookmark:addNewFolder()
     UIManager:show(path_chooser)
 end
 
-function FileManagerBookmark:addFolderFromInput(friendly_name, folder)
-    for __, item in ipairs(G_reader_settings:readSetting("favorite_folders") or {}) do
+function FileManagerShortcuts:addFolderFromInput(friendly_name, folder)
+    for __, item in ipairs(G_reader_settings:readSetting("shortcut_folders") or {}) do
         if item.text == friendly_name and item.folder == folder then
             UIManager:show(InfoMessage:new{
-                text = _("This folder already exist in favorites."),
+                text = _("This folder already exist in shortcuts."),
```

Comment:
`A shortcut to (for?) this folder already exists.`
`A shortcut with that name to (for?) this folder already exists.`